configuration/components/loki.libsonnet

```diff
@@ -485,6 +486,18 @@ function(params) {
     },
   },
 
+  rhobsLoki::
+    lokiMixins {
+      _config+:: {
+        multi_zone_ingester_enabled: false,
+        // Compactor
+        using_boltdb_shipper: true,
+        ruler_enabled: true,
+        query_scheduler_enabled: true,
+        use_index_gateway: true,
+      },
```

Comment:
Do you consider adding the [membelist_ring_enabled](https://github.com/grafana/loki/blob/5a08a6bcb9ecc816328cb7859bbdcc7cf9d004c5/production/ksonnet/loki/memberlist.libsonnet#L18) in this PR or a separate one? This options enables another k8s service for the gossip ring.